languages/pl-PL/icon.tid

```diff
@@ -0,0 +1,13 @@
+title: $:/languages/en-GB/icon
```

Comment:
I think this title is wrong. `en-GB` should probably be `pl-PL`

Reply:
Fixed, I missed it, thank you!